conf/application.conf

```diff
@@ -20,20 +20,33 @@ appName="residence-nil-rate-band-calculator-frontend"
 application.router=prod.Routes
 application.langs="en,cy"
 
-application.global=uk.gov.hmrc.residencenilratebandcalculator.FrontendGlobal
-
 play.filters.headers.contentSecurityPolicy= "default-src 'self' 'unsafe-inline' localhost:9250 localhost:9032 www.googletagmanager.com fonts.googleapis.com tagmanager.google.com ssl.gstatic.com www.gstatic.com fonts.gstatic.com www.google-analytics.com data:"
 
-# Play Modules
-# ~~~~
-# Additional play modules can be added here
-play.modules.enabled += "com.kenshoo.play.metrics.PlayModule"
+# An ApplicationLoader that uses Guice to bootstrap the application.
+play.application.loader = "uk.gov.hmrc.play.bootstrap.ApplicationLoader"
+
+# Primary entry point for all HTTP requests on Play applications
+play.http.requestHandler = "uk.gov.hmrc.play.bootstrap.http.RequestHandler"
+
+# Provides an implementation of AuditConnector. Use `uk.gov.hmrc.play.bootstrap.AuditModule` or create your own.
+# An audit connector must be provided.
+play.modules.enabled += "uk.gov.hmrc.play.bootstrap.AuditModule"
+
+# Provides an implementation of MetricsFilter. Use `uk.gov.hmrc.play.bootstrap.graphite.GraphiteMetricsModule` or create your own.
+# A metric filter must be provided
+play.modules.enabled += "uk.gov.hmrc.play.bootstrap.graphite.GraphiteMetricsModule"
+
+# Provides an implementation and configures all filters required by a Platform frontend microservice.
+play.modules.enabled += "uk.gov.hmrc.play.bootstrap.FrontendModule"
+play.http.filters = "uk.gov.hmrc.residencenilratebandcalculator.filters.Filters"
 
-# Global request handler
-# ~~~~
-# Set to the object handler for migrated 2.3 services
-play.http.requestHandler = "play.api.http.GlobalSettingsHttpRequestHandler"
+# Custom error handler
+play.http.errorHandler = "uk.gov.hmrc.residencenilratebandcalculator.utils.RnrbErrorHandler"
 
+cookie.encryption.key = "gvBoGdgzqG1AarzF1LY0zQ=="
+queryParameter.encryption.key = "gvBoGdgzqG1AarzF1LY0zQ=="
+sso.encryption.key = "gvBoGdgzqG1AarzF1LY0zQ=="
+play.crypto.secret="changethistosomethingsecret"
```

Comment:
What is this secret being set to/being overridden by?

Reply:
app-config-common